src/riderStatus.js

```diff
@@ -1,10 +1,8 @@
 /* eslint-disable indent */
 import protobuf from 'protobufjs';
 
-const extraVariables = ['roadID', 'rideOns', 'isTurning', 'isForward', 'cadence'];
-
-const proto = `syntax=\"proto3\";
-    message Status {
+const proto = `syntax="proto3";
```

Comment:
I think we want to move the protobuf definition out into another file (e.g. zwiftProtobuf.js), since it no longer represents just rider status.
This file can then import the "root" object and export the wrapper player status.